lib/class-wp-rest-menu-items-controller.php

```diff
@@ -516,135 +654,158 @@ public function get_item_schema() {
+				'sanitize_callback' => function ( $value ) {
+					return array_map( 'sanitize_html_class', explode( ' ', $value ) );
+				},
+			),
 		);
 
 		$schema['properties']['description'] = array(
 			'description' => __( 'The description of this menu item.' ),
-			'context'     => array( 'view', 'edit' ),
 			'type'        => 'string',
-		);
-
-		$schema['properties']['menu_item_parent'] = array(
-			'description' => __( 'The DB ID of the nav_menu_item that is this item\'s menu parent, if any . 0 otherwise . ' ),
-			'context'     => array( 'view', 'edit' ),
-			'type'        => 'integer',
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'arg_options' => array(
+				'sanitize_callback' => 'sanitize_text_field',
+			),
 		);
 
 		$schema['properties']['menu_order'] = array(
 			'description' => __( 'The DB ID of the nav_menu_item that is this item\'s menu parent, if any . 0 otherwise . ' ),
-			'context'     => array( 'view', 'edit' ),
+			'context'     => array( 'view', 'edit', 'embed' ),
 			'type'        => 'integer',
+			'minimum'     => 0,
+			'default'     => 0,
 		);
 		$schema['properties']['object']     = array(
 			'description' => __( 'The type of object originally represented, such as "category," "post", or "attachment."' ),
-			'context'     => array( 'view', 'edit' ),
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'type'        => 'string',
 		);
 
 		$schema['properties']['object_id'] = array(
 			'description' => __( 'The DB ID of the original object this menu item represents, e . g . ID for posts and term_id for categories .' ),
-			'context'     => array( 'view', 'edit' ),
+			'context'     => array( 'view', 'edit', 'embed' ),
 			'type'        => 'integer',
+			'minimum'     => 0,
+			'default'     => 0,
 		);
 
 		$schema['properties']['target'] = array(
-			'description' => __( 'The target attribute of the link element for this menu item . The family of objects originally represented, such as "post_type" or "taxonomy."' ),
-			'context'     => array( 'view', 'edit' ),
+			'description' => __( 'The target attribute of the link element for this menu item.' ),
 			'type'        => 'string',
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'enum'        => array(
+				'_blank',
+				'',
+			),
 		);
 
 		$schema['properties']['type_label'] = array(
 			'description' => __( 'The singular label used to describe this type of menu item.' ),
-			'context'     => array( 'view' ),
+			'context'     => array( 'view', 'embed' ),
 			'type'        => 'string',
 			'readonly'    => true,
 		);
 
 		$schema['properties']['url'] = array(
-			'description' => __( 'The URL to which this menu item points .' ),
+			'description' => __( 'The URL to which this menu item points.' ),
 			'type'        => 'string',
 			'format'      => 'uri',
-			'context'     => array( 'view', 'edit' ),
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'arg_options' => array(
+				'sanitize_callback' => 'esc_url_raw',
```

Comment:
Is there a reason you are applying this here instead of letting the REST API handle `uri` sanitization? It will call `esc_url_raw` via `rest_sanitize_value_from_schema`.

Reply:
Overkill, should I remove it?

Reply:
I think so.

Reply:
+1